[2018-11-12T16:10:39.917] [ERROR] dateFileLog - { Error: ER_BAD_FIELD_ERROR: Unknown column 'gradDate' in 'field list'
    at Query.Sequence._packetToError (D:\work\TOOLSWEB\node_modules\mysql\lib\protocol\sequences\Sequence.js:48:14)
    at Query.ErrorPacket (D:\work\TOOLSWEB\node_modules\mysql\lib\protocol\sequences\Query.js:82:18)
    at Protocol._parsePacket (D:\work\TOOLSWEB\node_modules\mysql\lib\protocol\Protocol.js:271:23)
    at Parser.write (D:\work\TOOLSWEB\node_modules\mysql\lib\protocol\Parser.js:77:12)
    at Protocol.write (D:\work\TOOLSWEB\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (D:\work\TOOLSWEB\node_modules\mysql\lib\Connection.js:82:28)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    --------------------
    at Protocol._enqueue (D:\work\TOOLSWEB\node_modules\mysql\lib\protocol\Protocol.js:135:48)
    at PoolConnection.Connection.query (D:\work\TOOLSWEB\node_modules\mysql\lib\Connection.js:185:25)
    at doQuery (D:\work\TOOLSWEB\yujiang.Foil.Node\src\yjDBService.engine.mysql.js:474:24)
    at success (D:\work\TOOLSWEB\yujiang.Foil.Node\src\yjDBService.engine.mysql.js:465:9)
    at Object.exports.safeSuccess (D:\work\TOOLSWEB\yujiang.Foil.Node\src\yjError.js:94:13)
    at Object.exports.handleResult (D:\work\TOOLSWEB\yujiang.Foil.Node\src\yjError.js:148:12)
    at D:\work\TOOLSWEB\yujiang.Foil.Node\src\yjDBService.engine.mysql.js:89:11
    at Ping.onPing (D:\work\TOOLSWEB\node_modules\mysql\lib\Pool.js:94:5)
    at bound (domain.js:301:14)
    at Ping.runBound [as _callback] (domain.js:314:12)
  code: 'ER_BAD_FIELD_ERROR',
  errno: 1054,
  sqlState: '42S22',
  index: 0,
  sql: 'insert into OA_staffs (staff_ID,dept,deptClass,deptGroup,staffName,sex,deptFee,postName,postLevel,edu,company,birthday,entryDate,inUnion,Phone,major,school,degree,certificates,gradDate) values (\'\',\'王三\',\'男\',\'\',\'\',\'\',\'\',\'\',\'\',\'\',\'\',\'\',\'\',\'\',\'\',\'\',\'\',\'\',\'\',\'\')' }
[2018-11-12T16:10:39.924] [ERROR] dateFileLog - Error: ER_BAD_FIELD_ERROR: Unknown column 'gradDate' in 'field list'
    at D:\work\TOOLSWEB\yujiang.Foil.Node\src\yjREST.engine.remote.superagent.js:43:34
    at Request.callback (D:\work\TOOLSWEB\node_modules\superagent\lib\node\index.js:797:3)
    at IncomingMessage.<anonymous> (D:\work\TOOLSWEB\node_modules\superagent\lib\node\index.js:990:12)
    at emitNone (events.js:111:20)
    at IncomingMessage.emit (events.js:208:7)
    at endReadableNT (_stream_readable.js:1064:12)
    at _combinedTickCallback (internal/process/next_tick.js:139:11)
    at process._tickDomainCallback (internal/process/next_tick.js:219:9)
[2018-11-12T16:10:54.114] [ERROR] dateFileLog - { Error: ER_BAD_FIELD_ERROR: Unknown column 'gradDate' in 'field list'
    at Query.Sequence._packetToError (D:\work\TOOLSWEB\node_modules\mysql\lib\protocol\sequences\Sequence.js:48:14)
    at Query.ErrorPacket (D:\work\TOOLSWEB\node_modules\mysql\lib\protocol\sequences\Query.js:82:18)
    at Protocol._parsePacket (D:\work\TOOLSWEB\node_modules\mysql\lib\protocol\Protocol.js:271:23)
    at Parser.write (D:\work\TOOLSWEB\node_modules\mysql\lib\protocol\Parser.js:77:12)
    at Protocol.write (D:\work\TOOLSWEB\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (D:\work\TOOLSWEB\node_modules\mysql\lib\Connection.js:82:28)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    --------------------
    at Protocol._enqueue (D:\work\TOOLSWEB\node_modules\mysql\lib\protocol\Protocol.js:135:48)
    at PoolConnection.Connection.query (D:\work\TOOLSWEB\node_modules\mysql\lib\Connection.js:185:25)
    at doQuery (D:\work\TOOLSWEB\yujiang.Foil.Node\src\yjDBService.engine.mysql.js:474:24)
    at success (D:\work\TOOLSWEB\yujiang.Foil.Node\src\yjDBService.engine.mysql.js:465:9)
    at Object.exports.safeSuccess (D:\work\TOOLSWEB\yujiang.Foil.Node\src\yjError.js:94:13)
    at Object.exports.handleResult (D:\work\TOOLSWEB\yujiang.Foil.Node\src\yjError.js:148:12)
    at D:\work\TOOLSWEB\yujiang.Foil.Node\src\yjDBService.engine.mysql.js:89:11
    at Ping.onPing (D:\work\TOOLSWEB\node_modules\mysql\lib\Pool.js:94:5)
    at bound (domain.js:301:14)
    at Ping.runBound [as _callback] (domain.js:314:12)
  code: 'ER_BAD_FIELD_ERROR',
  errno: 1054,
  sqlState: '42S22',
  index: 0,
  sql: 'insert into OA_staffs (staff_ID,dept,deptClass,deptGroup,staffName,sex,deptFee,postName,postLevel,edu,company,birthday,entryDate,inUnion,Phone,major,school,degree,certificates,gradDate) values (\'001\',\'王三\',\'男\',\'\',\'\',\'\',\'\',\'\',\'\',\'\',\'\',\'\',\'\',\'\',\'\',\'\',\'\',\'\',\'\',\'\')' }
[2018-11-12T16:10:54.118] [ERROR] dateFileLog - Error: ER_BAD_FIELD_ERROR: Unknown column 'gradDate' in 'field list'
    at D:\work\TOOLSWEB\yujiang.Foil.Node\src\yjREST.engine.remote.superagent.js:43:34
    at Request.callback (D:\work\TOOLSWEB\node_modules\superagent\lib\node\index.js:797:3)
    at IncomingMessage.<anonymous> (D:\work\TOOLSWEB\node_modules\superagent\lib\node\index.js:990:12)
    at emitNone (events.js:111:20)
    at IncomingMessage.emit (events.js:208:7)
    at endReadableNT (_stream_readable.js:1064:12)
    at _combinedTickCallback (internal/process/next_tick.js:139:11)
    at process._tickDomainCallback (internal/process/next_tick.js:219:9)
[2018-11-12T16:22:44.308] [ERROR] dateFileLog - { Error: ER_BAD_FIELD_ERROR: Unknown column 'gradDate' in 'field list'
    at Query.Sequence._packetToError (D:\work\TOOLSWEB\node_modules\mysql\lib\protocol\sequences\Sequence.js:48:14)
    at Query.ErrorPacket (D:\work\TOOLSWEB\node_modules\mysql\lib\protocol\sequences\Query.js:82:18)
    at Protocol._parsePacket (D:\work\TOOLSWEB\node_modules\mysql\lib\protocol\Protocol.js:271:23)
    at Parser.write (D:\work\TOOLSWEB\node_modules\mysql\lib\protocol\Parser.js:77:12)
    at Protocol.write (D:\work\TOOLSWEB\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (D:\work\TOOLSWEB\node_modules\mysql\lib\Connection.js:82:28)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    --------------------
    at Protocol._enqueue (D:\work\TOOLSWEB\node_modules\mysql\lib\protocol\Protocol.js:135:48)
    at PoolConnection.Connection.query (D:\work\TOOLSWEB\node_modules\mysql\lib\Connection.js:185:25)
    at doQuery (D:\work\TOOLSWEB\yujiang.Foil.Node\src\yjDBService.engine.mysql.js:474:24)
    at success (D:\work\TOOLSWEB\yujiang.Foil.Node\src\yjDBService.engine.mysql.js:465:9)
    at Object.exports.safeSuccess (D:\work\TOOLSWEB\yujiang.Foil.Node\src\yjError.js:94:13)
    at Object.exports.handleResult (D:\work\TOOLSWEB\yujiang.Foil.Node\src\yjError.js:148:12)
    at D:\work\TOOLSWEB\yujiang.Foil.Node\src\yjDBService.engine.mysql.js:89:11
    at Ping.onPing (D:\work\TOOLSWEB\node_modules\mysql\lib\Pool.js:94:5)
    at bound (domain.js:301:14)
    at Ping.runBound [as _callback] (domain.js:314:12)
  code: 'ER_BAD_FIELD_ERROR',
  errno: 1054,
  sqlState: '42S22',
  index: 0,
  sql: 'insert into OA_staffs (staff_ID,dept,deptClass,deptGroup,staffName,sex,deptFee,postName,postLevel,edu,company,birthday,entryDate,inUnion,Phone,major,school,degree,certificates,gradDate) values (\'111\',\'软体部\',\'突突突\',\'突突突\',\' 突突突\',\'男\',\'100\',\'突突突\',\'突突突\',\'突突突\',\'突突突\',\'2018-10-11\',\'2018-10-11\',\'是\',\'1123123\',\'钉钉钉钉\',\'对对对 钉钉\',\'对对对\',\'对对对,对对对\',\'2018-10\')' }
[2018-11-12T16:22:44.316] [ERROR] dateFileLog - Error: ER_BAD_FIELD_ERROR: Unknown column 'gradDate' in 'field list'
    at D:\work\TOOLSWEB\yujiang.Foil.Node\src\yjREST.engine.remote.superagent.js:43:34
    at Request.callback (D:\work\TOOLSWEB\node_modules\superagent\lib\node\index.js:797:3)
    at IncomingMessage.<anonymous> (D:\work\TOOLSWEB\node_modules\superagent\lib\node\index.js:990:12)
    at emitNone (events.js:111:20)
    at IncomingMessage.emit (events.js:208:7)
    at endReadableNT (_stream_readable.js:1064:12)
    at _combinedTickCallback (internal/process/next_tick.js:139:11)
    at process._tickDomainCallback (internal/process/next_tick.js:219:9)
[2018-11-12T16:23:45.896] [ERROR] dateFileLog - { Error: ER_BAD_FIELD_ERROR: Unknown column 'gradDate' in 'field list'
    at Query.Sequence._packetToError (D:\work\TOOLSWEB\node_modules\mysql\lib\protocol\sequences\Sequence.js:48:14)
    at Query.ErrorPacket (D:\work\TOOLSWEB\node_modules\mysql\lib\protocol\sequences\Query.js:82:18)
    at Protocol._parsePacket (D:\work\TOOLSWEB\node_modules\mysql\lib\protocol\Protocol.js:271:23)
    at Parser.write (D:\work\TOOLSWEB\node_modules\mysql\lib\protocol\Parser.js:77:12)
    at Protocol.write (D:\work\TOOLSWEB\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (D:\work\TOOLSWEB\node_modules\mysql\lib\Connection.js:82:28)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    --------------------
    at Protocol._enqueue (D:\work\TOOLSWEB\node_modules\mysql\lib\protocol\Protocol.js:135:48)
    at PoolConnection.Connection.query (D:\work\TOOLSWEB\node_modules\mysql\lib\Connection.js:185:25)
    at doQuery (D:\work\TOOLSWEB\yujiang.Foil.Node\src\yjDBService.engine.mysql.js:474:24)
    at success (D:\work\TOOLSWEB\yujiang.Foil.Node\src\yjDBService.engine.mysql.js:465:9)
    at Object.exports.safeSuccess (D:\work\TOOLSWEB\yujiang.Foil.Node\src\yjError.js:94:13)
    at Object.exports.handleResult (D:\work\TOOLSWEB\yujiang.Foil.Node\src\yjError.js:148:12)
    at D:\work\TOOLSWEB\yujiang.Foil.Node\src\yjDBService.engine.mysql.js:89:11
    at Ping.onPing (D:\work\TOOLSWEB\node_modules\mysql\lib\Pool.js:94:5)
    at bound (domain.js:301:14)
    at Ping.runBound [as _callback] (domain.js:314:12)
  code: 'ER_BAD_FIELD_ERROR',
  errno: 1054,
  sqlState: '42S22',
  index: 0,
  sql: 'insert into OA_staffs (staff_ID,dept,deptClass,deptGroup,staffName,sex,deptFee,postName,postLevel,edu,company,birthday,entryDate,inUnion,Phone,major,school,degree,certificates,gradDate) values (\'111\',\'软体部\',\'突突突\',\'突突突\',\' 突突突\',\'男\',\'100\',\'突突突\',\'突突突\',\'突突突\',\'突突突\',\'2018-10-11\',\'2018-10-11\',\'是\',\'1123123\',\'钉钉钉钉\',\'对对对 钉钉\',\'对对对\',\'对对对,对对对\',\'2018-10\')' }
[2018-11-12T16:23:45.899] [ERROR] dateFileLog - Error: ER_BAD_FIELD_ERROR: Unknown column 'gradDate' in 'field list'
    at D:\work\TOOLSWEB\yujiang.Foil.Node\src\yjREST.engine.remote.superagent.js:43:34
    at Request.callback (D:\work\TOOLSWEB\node_modules\superagent\lib\node\index.js:797:3)
    at IncomingMessage.<anonymous> (D:\work\TOOLSWEB\node_modules\superagent\lib\node\index.js:990:12)
    at emitNone (events.js:111:20)
    at IncomingMessage.emit (events.js:208:7)
    at endReadableNT (_stream_readable.js:1064:12)
    at _combinedTickCallback (internal/process/next_tick.js:139:11)
    at process._tickDomainCallback (internal/process/next_tick.js:219:9)
